doc/swagger-api-description/api.yaml

```diff
@@ -371,6 +578,202 @@ definitions:
+          $ref: "#/definitions/VufsFile"
+      allocateModel:
+        $ref: "#/definitions/AllocationModel"
+
+  Property:
+    description: Describes all file property.
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Property UUID.
+        format: uuid
+      property-name:
+        type: string
+        description: Property name.
+      property-value:
+        type: string
+        description: Property value.
+
+  PartLocation:
+    description: Describes sequence number of file part and its location.
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: PartLocation UUID.
+        format: uuid
+      number:
+        type: integer
+        format: int32
+        description: Sequence number.
+      location:
+        $ref: "#/definitions/Cloud"
+
+  AllocationModel:
+    description: Describes allocate storage model ot file.
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Allocation model UUID.
+        format: uuid
+      modelType:
+        description: Types of allocation model.
+        enum:
+          - UNION
+          - MIRROR
+          - HIGH_AVAILABILITY
+      propertySet:
+        type: array
+        items:
+          $ref: "#/definitions/Property"
+
+  InitialAllocationModel:
+    type: object
+    description: Contain information about initial file allocation stategy.
+    properties:
+      uuid:
+        type: string
+        description: InitialAllocateModel UUID.
+        format: uuid
+      file:
+        $ref: "#/definitions/VufsFile"
+      allocateModel:
+        $ref: "#/definitions/AllocationModel"
+
+  StatisticSet:
+    type: object
+    description: Contain set of statistic information about cloud.
+    properties:
+      statisticSet:
+        type: array
+        items:
+          $ref: "#/definitions/Statistic"
+
+  Statistic:
+    type: object
+    description: Contain statistic information about cloud.
+    properties:
+      uuid:
+        type: string
+        description: Statistic UUID.
+        format: uuid
+      cloud:
+        $ref: "#/definitions/Cloud"
+      uploadFiles:
+        description: Count of uploaded files.
+        type: integer
+        format: int32
+      uploadTime:
+        description: Time what files was uploaded.
+        type: integer
+        format: int64
+      downloadFiles:
+        description: Count of downloaded files.
+        type: integer
+        format: int32
+      downloadTimes:
+        description: Time what files was downloaded.
+        type: integer
+        format: int64
+
+  HelthStatistis:
+    type: object
+    description: Contain helth-check statistic information about set of clouds.
+    properties:
+      uuid:
+        type: string
+        description: HelthStatistis UUID.
+        format: uuid
+      helthCloudSet:
+        type: array
+        items:
+          $ref: "#/definitions/HelthCloud"
+
+  HelthCloud:
```

Comment:
Remove this object. Use Statistic and StatisticSet object with keys-value:
[{uuid: 'clouds's uuid'}, {available: true}]